tests/boot/snapper_rollback.pm

```diff
@@ -22,7 +22,7 @@ use version_utils;
 
 sub run {
     my ($self) = @_;
-    if (is_leap_migration && (check_var('DESKTOP', 'gnome') || check_var('DESKTOP', 'kde'))) {
+    if ((is_leap_migration && (check_var('DESKTOP', 'gnome') || check_var('DESKTOP', 'kde'))) || is_opensuse) {
```

Comment:
So you consider opensuse without desktop test?

Reply:
It just means we boot direct into desktop without display manager in opensuse.

Reply:
according the condition, is_opensuse will assert the displaymanager, so are you confirm there is no textmode for opensuse migration test?

Reply:
changed and verified with test run.